js/pihole/queryads.js

```diff
@@ -1,38 +1,77 @@
+var exact = "";
 function eventsource() {
     var ta = $("#output");
     var domain = $("#domain");
+    var q = $("#quiet");
     if(domain.val().length === 0)
     {
         return;
     }
-    var source = new EventSource("php/queryads.php?domain="+domain.val());
+
+    var quiet = false;
+    if(q.val() === "yes")
+    {
+        quiet = true;
+    }
+
+    if(quiet)
+    {
+        exact = "exact";
+    }
+
+    var host = window.location.host;
+    var source = new EventSource("http://"+host+"/admin/php/queryads.php?domain="+domain.val()+"&"+exact);
```

Comment:
Does it matter what host the php is called from? We can replace `"+host+"` with `pi.hole` I think since the user should be able to reach that address (since they are getting the block page).